StripePaymentSheet/StripePaymentSheet/Source/Helpers/PaymentSheetLinkAccount.swift

```diff
@@ -452,38 +460,9 @@ private extension PaymentSheetLinkAccount {
     }
 
     func refreshSession(
-        completion: @escaping (Result<Void, Error>) -> Void
+        completion: @escaping (Result<ConsumerSession, Error>) -> Void
     ) {
-        // The consumer session lookup endpoint currently serves as our endpoint for
-        // refreshing the session. To refresh the session, we need to call this endpoint
-        // without providing an email address.
-        ConsumerSession.lookupSession(
-            for: nil,  // No email address
-            emailSource: nil, // No source
-            sessionID: elementsSessionID,
-            with: apiClient,
-            useMobileEndpoints: useMobileEndpoints
-        ) { [weak self] result in
-            switch result {
-            case .success(let response):
-                switch response.responseType {
-                case .found(let session):
-                    self?.currentSession = session.consumerSession
-                    self?.publishableKey = session.publishableKey
-                    completion(.success(()))
-                case .notFound(let message):
-                    completion(
-                        .failure(PaymentSheetError.linkLookupNotFound(serverErrorMessage: message))
-                    )
-                case .noAvailableLookupParams:
-                    completion(
-                        .failure(PaymentSheetError.missingClientSecret)
-                    )
-                }
-            case .failure(let error):
-                completion(.failure(error))
-            }
-        }
+        paymentSheetLinkAccountDelegate?.refreshLinkSession(completion: completion)
```

Comment:
Seems like if `paymentSheetLinkAccountDelegate` is not set, the completion will not get called.  Seems like we should add a guard statement like..
```
guard let delegate = paymentSheetLinkAccountDelegate {
   stpAssertionFailure()
   completion(failure)
   return 
}
delegate.refreshLinkSession(completion:completion)
```